packages/suite/src/components/suite/copy/IOAddress.tsx

```diff
@@ -55,15 +55,9 @@ const TextOverflowContainer = styled.div<{ $shouldAllowCopy?: boolean }>`
         `}
 `;
 
-const SpanTextStart = styled.span`
-    display: inline-block;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-`;
-
-const SpanTextEnd = styled.span`
-    display: inline-block;
+const BreakableTextDiv = styled.div`
+    word-break: break-all;
+    padding: 0 1px;
```

Comment:
And I'm not sure what for do we need horizontal padding with 1px 🤔

Reply:
there was sa cut at the end of the line for some reason